app/src/main/java/com/example/footprnt/HomeActivity.java

```diff
@@ -51,6 +51,9 @@ public void onActivityResult(int requestCode, int resultCode, Intent data) {
         if(resultCode == 302){
             mFragment3.onActivityResult(requestCode, resultCode, data);
         }
+        if(resultCode == 301){
```

Comment:
I want you to focus on consistency in your spacing. IF blocks shouldn't be "if(something){" while methods are "void methodName() {". Use a space between the parantheses and brackets or don't